math/fibonacci.cpp

```diff
@@ -13,12 +13,15 @@
 
 /**
  * Recursively compute sequences
+ * @param n input
+ * @returns n-th element of the Fbinacci's sequence
  */
 unsigned int fibonacci(unsigned int n) {
```

Comment:
could you update the code to use `uint64_t` instead of `int` and `unsigned int`.
Thanks

Reply:
done